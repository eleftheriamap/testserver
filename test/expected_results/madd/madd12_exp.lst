
test/exp/test/gen/madd/madd12_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	514f5318 	sub	w24, w24, #0x3d4, lsl #12
   4:	1111f8a5 	add	w5, w5, #0x47e
   8:	516ebd6b 	sub	w11, w11, #0xbaf, lsl #12
   c:	1b052f16 	madd	w22, w24, w5, w11
  10:	8a000000 	and	x0, x0, x0
