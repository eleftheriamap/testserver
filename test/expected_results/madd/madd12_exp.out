Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 000000000000047e
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = 0000000000000000
X10    = 0000000000000000
X11    = 00000000ff451000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 0000000000000000
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 00000000ec1f9000
X23    = 0000000000000000
X24    = 00000000ffc2c000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 0000000000000000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 0000000000000010
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x514f5318
0x00000004 : 0x1111f8a5
0x00000008 : 0x516ebd6b
0x0000000c : 0x1b052f16
0x00000010 : 0x8a000000
